ceph.spec.in

```diff
@@ -159,7 +164,15 @@ BuildRequires:	gperftools-devel >= 2.6.1
 BuildRequires:	gperftools-devel >= 2.4
 %endif
 %endif
+%if 0%{with qa_run}
 BuildRequires:  jq
```

Comment:
@tchaikov Are we sure `jq` is not a (non-interactive) `make check` dependency? I see it being used, e.g., in `src/test/cli/crushtool/choose-args.t`

Reply:
fixed.